### DISCUSSION TITLE
play dsd stutter on ubuntu 22 or 24 LTS,debain is OK

### DISCUSSION THREAD

**Discussion by YTSamLee** [MusicPlayerDaemon/MPD#2129] [Q&A]
I use mpd+upmpdcli to build upnp renderer on Raspberry Pi or X86 low-power machine.

PI 4 is ok on debian 12 Whether it is playing PCM or DSD;
==================
upboard or dell wyse 3040(cpu is Intel Atom® x5-Z8350) on ubuntu 22 or 24lts, playing DSD is stuttered, playing PCM is ok.
On Ubuntu 22 or 24, whether the stock kernel or other low-latency or real-time kernel, playing DSD is always stuttered after a few seconds。
I want to know, can this stuttering be solved in Ubuntu?

**Comment by MaxKellermann** [comment]
https://www.musicpd.org/help/
